spec/models/student_loans/school_eligibility_spec.rb

```diff
@@ -99,6 +171,16 @@
         context "and it doesn't have an education phase" do
           let(:school_attributes) { special_free_school_attributes.merge({phase: :not_applicable}) }
           it { is_expected.to be true }
+
+          context "and it closed after the start of the policy" do
+            let(:school_attributes) { special_free_school_attributes.merge({phase: :not_applicable, close_date: after_start_of_policy}) }
+            it { is_expected.to be true }
+          end
+
+          context "and it closed before the start of the policy" do
+            let(:school_attributes) { special_free_school_attributes.merge({phase: :not_applicable, close_date: before_start_of_policy}) }
+            it { is_expected.to be false }
```

Comment:
Do all these repeating contexts gives us extra confidence the check method is working correctly, I found the repatition made me lazy reading the tests?

Reply:
Yeah I think we wanted to build up a matrix of all the possibilities for eligibility criteria. I think it helps make clear that the school in each example would otherwise be eligible if it weren't for it closing before the start of the policy.